Type Definition Error TS2309

First off, thanks for putting this library together! I noticed this error while developing today: 

```
node_modules/initials/index.d.ts:20:2 - error TS2309: An export assignment cannot be used in a module with other exported elements.

  export = initials;
  ~~~~~~~~~~~~~~~~~~
```

Changing this line to `export default initials` fixes the error.

TypeScript version: 3.9.5
initials version: 3.1.0

I'm happy to open a follow-up pull request.

--- a/index.d.ts
+++ b/index.d.ts
@@ -17,5 +17,5 @@ declare module 'initials' {
 
 	export function addTo(name: string): string | undefined;
 
-	export = initials;
+	export default initials;
 }

